react-native-reanimated.d.ts

```diff
@@ -357,7 +357,7 @@ declare module 'react-native-reanimated' {
         inputRange: ReadonlyArray<Adaptable<number>>;
         outputColorRange: (string | number)[];
```

Comment:
Could you also change this type to `ReadonlyArray<Adaptable<number | string>>`, like in [`interpolateNode`](https://github.com/software-mansion/react-native-reanimated/pull/1467/files#diff-3eb03eea3c56460a1bdcef23b6d54e490742744b667bb3ebacc1927b8f7d2b49R57)?

Reply:
sure thing.